src/components/Header.js

```diff
@@ -1,11 +1,40 @@
 import React, { Component } from "react";
+import { Nav, NavItem, NavLink } from 'reactstrap';
+import { Jumbotron, Container } from 'reactstrap';
 
 class Header extends Component {
 
     render () {
         return (
             <>
-            <h1>Header </h1>
+            <div>
```

Comment:
Will you double check your indentation here? It might just be my view, but make sure all child tags are nested one tab and the closing tags match up.